tests/framework/microvm.py

```diff
@@ -1001,7 +1001,7 @@ def wait_for_up(self, timeout=10):
         :param timeout: seconds to wait.
         """
         try:
-            rc, stdout, stderr = self.ssh.run("true", timeout)
+            rc, stdout, stderr = self.ssh.run("true", timeout, warn_on_fail=False)
```

Comment:
TBH I think those messages are fine. We can print a message saying "timeout messages are expected while the microvm boots".

Reply:
They're quite spammy tbh. The one in `init_connection` has 20 retries, and if it does need all or most of them, then scrolling through them gets old quite fast :sweat_smile:

Reply:
For example we could change `_init_connection` to:

```
        wait=wait_chain(wait_fixed(.5), wait_fixed(.3)),
        stop=stop_after_delay(3),
```

This would wait .5s on the 1st attempt, then .3s until we reach 3s.

Reply:
Alright, I've dropped the commit, since it is kinda orthogonal to the main goal of this PR anyway, maybe can revisit the logging in another context and solve it properly :)